tools/docker.sh

```diff
@@ -2,7 +2,37 @@
 
 set -e
 
-for tag in latest $(git tag --points-at | sed s/^v//); do
-  docker build -t raviqqe/muffet:$tag .
-  docker push raviqqe/muffet:$tag
-done
+check_buildx(){
+    if docker buildx version &>/dev/null; then
+        return 0
+    fi
+    if docker buildx ls &>/dev/null; then
+        return 0
+    fi
+    return 1
+}
```

Comment:
When it comes to make sure if buildx is installed I'd recommend you this:

```suggestion
check_buildx(){
    docker buildx &>/dev/null
    return @?
}
```